gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/GraphTraversal.java

```diff
@@ -331,6 +331,21 @@ public default GraphTraversal<S, Vertex> V(final Object... vertexIdsOrElements)
         return this.asAdmin().addStep(new GraphStep<>(this.asAdmin(), Vertex.class, false, ids));
     }
 
+    /**
+     * A {@code E} step is usually used to start a traversal but it may also be used mid-traversal.
+     *
+     * @param edgeIdsOrElements vertices to inject into the traversal
```

Comment:
"vertices" should be "edges" in the javadoc there.

Reply:
```suggestion
     * @param edgeIdsOrElements edges to inject into the traversal
```

Reply:
fixed in https://github.com/apache/tinkerpop/pull/1808/commits/13501c50233d1692d2cd3dd8185e61e9050f8fe3